src/main/java/mainclass/exercises/streams/FavoriteBook.java

```diff
@@ -0,0 +1,24 @@
+package mainclass.exercises.streams;
+
+public class FavoriteBook {
+    String title;
+    String author;
+    String genre;
+    int year;
```

Comment:
Use the private access modifier, please.

Reply:
updated, thank you!